R/BoxCox.R

```diff
@@ -33,9 +33,14 @@
 #'
 #' # Tidying
 #'
-#' When you [`tidy()`][tidy.recipe()] this step, a tibble with columns
-#' `terms` (the selectors or variables selected) and `value` (the
-#' lambda estimate) is returned.
+#' When you [`tidy()`][tidy.recipe()] this step, a tibble returned with 3
```

Comment:
```suggestion
#' When you [`tidy()`][tidy.recipe()] this step, a tibble is returned with
```

I would recommend this change in all places where it occurs.

Reply:
Done in a0e9d3922